src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -423,4 +423,9 @@ def replace_jquery_meta_characters(input)
     # The stated characters are c&p from https://api.jquery.com/category/selectors/
     input.gsub(/[!"#$%&'()*+,.\/:\\;<=>?@\[\]^`{|}~]/, '_')
   end
+
+  def word_break(string, length)
+    # adds a <wbr> tag after an amount of given characters
+    string.scan(/.{1,#{length}}/).join("<wbr>")
```

Comment:
Does that work? I would assume that you need to mark ``.join("<wbr>")`` as ``html_safe``